wfsim/core/s2.py

```diff
@@ -43,15 +43,16 @@ def __call__(self, instruction):
         else:
             z_obs, positions = z, np.array([x, y]).T
 
+        n_electron = self.get_electron_yield(n_electron=n_electron,
```

Comment:
Could you change also the docstring of this function where it says "observed"? Also, for readability it would be better to change from z_obs to z